tests/behat/src/FixtureContext.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace SilverStripe\SessionManager\Tests\Behat\Context;
+
+use Behat\Mink\Element\NodeElement;
+use SilverStripe\BehatExtension\Context\FixtureContext as BaseFixtureContext;
+use SilverStripe\Control\Controller;
+use SilverStripe\Security\Member;
+use SilverStripe\SessionManager\Model\LoginSession;
+
+/**
+ * Context used to create fixtures in the SilverStripe ORM.
+ */
+class FixtureContext extends BaseFixtureContext
+{
+    /**
+     * @When /^I see the "([^"]+)" element$/
+     * @param $selector
+     */
+    public function iSeeTheElement($selector): void
+    {
+        $page = $this->getMainContext()->getSession()->getPage();
+        $element = $page->find('css', $selector);
+        assertNotNull($element, sprintf('Element %s not found', $selector));
+    }
+
+    /**
+     * @When /^I click the "([^"]+)" element$/
+     * @param $selector
+     */
+    public function iClickTheElement(string $selector): void
```

Comment:
You might be able to use [I press the *** button](https://github.com/silverstripe/silverstripe-behat-extension/blob/fe0818afc293d4c708d9cb075871fe9267d6326f/src/Context/BasicContext.php#L438-L443) or [I click in the element](https://github.com/silverstripe/silverstripe-behat-extension/blob/fe0818afc293d4c708d9cb075871fe9267d6326f/src/Context/BasicContext.php#L490-L509)

Reply:
Again, I think it's fine to add this here, and I've run into this issue several times before of not being able to click an element and each time I added this same function
- https://github.com/silverstripe/silverstripe-asset-admin/blob/1/tests/behat/src/FixtureContext.php#L360
- https://github.com/silverstripe/silverstripe-campaign-admin/blob/1/tests/behat/src/FixtureContext.php#L239
- https://github.com/silverstripe/silverstripe-userforms/blob/5/tests/behat/src/FixtureContext.php#L22

Probably should've added this pretty basic function to behat-extension back in the day, probably a bit late now

Reply:
If it's been a recurring problem than we should fix it upstream.

Reply:
Agree that's the best outcome.  I don't think it's an easy fix though and I think we should still add it here.  I've created a new issues to get it into behat-extension https://github.com/silverstripe/silverstripe-behat-extension/issues/200